### PR TITLE
feature/cplp-1732-document-enum-db-adjustment

### PR DESCRIPTION
**Pull request by qxz2mqe**
https://jira.catena-x.net/browse/CPLP-1732

### CODE REVIEW

**Comment by jjeroch** [comment]
@qxz2mqe we need to do an update in devenv as well. Have you already considered that?

**Comment by qxz2mqe** [comment]
yes

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (Phil91) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.cs:1]: Please add the file header to the generated files

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.Designer.cs [hunk 2]
```diff
@@ -0,0 +1,6010 @@
+﻿// <auto-generated />
```
Comment (Phil91) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.Designer.cs:1]: Please add the file header to the generated files

File: src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs [hunk 3]
```diff
@@ -1,24 +1,4 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- ********************************************************************************/
-
-// <auto-generated />
+﻿// <auto-generated />
```
Comment (Phil91) [inline src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs:1]: Please readd the file header

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.cs (hunk 1 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.cs]: fixed

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.Designer.cs (hunk 2 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/20221104115911_CPLP-1732-ChangeDocumentTypeEnum.Designer.cs]: fixed

File: src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs (hunk 3 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs]: fixed

**Comment by Phil91** [comment]
@qxz2mqe as Julia mentioned please create a PR in devenv for your database changes as well. Therefor you need to run `backup_portal_basedata.sh` & `backup_portal_data.sh` against your updated database.

**Review by ntruchsess — APPROVED** [review]

**Review by Phil91 — APPROVED** [review]
As soon as devenv is ready, this PR is ready for merging

**Pull request merged**